solutions/observability/get-started/opentelemetry/_snippets/edot-collector-auth.md

```diff
@@ -0,0 +1,22 @@
+```yaml
+extensions:
+  bearertokenauth:
+    scheme: "APIKey"
+    token: "<ENCODED_ELASTICSEARCH_APIKEY>"
+
+  apmconfig:
+    opamp:
+      protocols:
+        http:
+          # Default is localhost:4320
+          # endpoint: "<CUSTOM_OPAMP_ENDPOINT>"
+    source:
+      elasticsearch:
+        endpoint: "<ELASTICSEARCH_ENDPOINT>"
+        auth:
+          authenticator: bearertokenauth
+```
+
+:::{note}
+For comprehensive authentication configuration options, see [Authentication methods](/reference/edot-collector/config/authentication-methods.md).
```

Comment:
Is this supposed to be this? I wasn't sure because I don't think this snippet is being used anywhere. 

```suggestion
For comprehensive authentication configuration options, see [Authentication methods](elastic-agent://reference/edot-collector/config/authentication-methods.md).
```